recipes/xorg-proto/all/test_package/conanfile.py

```diff
@@ -0,0 +1,17 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "arch", "compiler", "build_type"
+    generators = "cmake", "pkg_config"
```

Comment:
Unless we want to test the generated pc files?
```suggestion
    generators = "cmake"
```